Doc/howto/logging.rst

```diff
@@ -89,9 +89,8 @@ described below (in increasing order of severity):
 |              | itself may be unable to continue running.   |
 +--------------+---------------------------------------------+
 
-The default level is ``WARNING``, which means that only events of this level
-and above will be tracked, unless the logging package is configured to do
-otherwise.
+The default level is ``WARNING``, which means that only events of this level and higher severity and above will be tracked, 
```

Comment:
Would prefer the wording "only events of this severity and higher will be tracked" here.